assets/js/analyzer.js

```diff
@@ -26,7 +26,9 @@ if(modeEnabled('analyzation')) {
         $('#morphAnalyzerInput').keydown(function (e) {
             if(e.keyCode === ENTER_KEY_CODE && !e.shiftKey) {
                 e.preventDefault();
-                analyze();
+                if($('#morphAnalyzerInput').val().trim() !== '') {
```

Comment:
Ohh. Awesome. I just now figured that submitting the forms with nothing but spaces by clicking on button also gives the error. Shifting this validation there would fix that error too! 

![screenshot from 2017-08-18 09-41-00](https://user-images.githubusercontent.com/8894636/29443961-5df6fc52-83f9-11e7-9000-228bea1da5be.png)

I submit the form with spaces and then it gives error.